Menu/GameShell/10_Settings/Bluetooth/__init__.py

```diff
@@ -1,16 +1,15 @@
-# -*- coding: utf-8 -*- 
+# -*- coding: utf-8 -*-
 import re
 import pygame
 #import math
 import  commands
 import dbus
-from beeprint import pp
```

Comment:
This is fixed https://github.com/panyanyany/beeprint/issues/26 (can be left there, but I'm not sure I understand what it does).

Reply:
pp is for the pretty print the debug info,  I use it in the process of developing , but I left it there cause I thought it is harmless